src/__tests__/extractMeta.js

```diff
@@ -2,6 +2,9 @@ import extractMetadata from '../extractMeta';
 import ElementPropTypes from '../propTypes';
 
 describe('Metadata extractor', () => {
+    it('fails gracefully when given bad input', () => {
+        expect(() => extractMetadata()).not.toThrow();
```

Comment:
I wonder where did you experienced getting no schema.

Reply:
@santios-volusion It's in the SiteDesigner, when adding a new block to a page, and attempting to extract the component metadata, to set the initial block config. The Site Designer is not correctly differentiating between new-contract blocks, which have `configSchema` as a top-level export, and old-contract blocks, which do not.